py3status/modules/nvidia_temp.py

```diff
@@ -43,33 +59,21 @@ def nvidia_temp(self):
         if temps != []:
             data = []
             for temp in temps:
-                fmt_str = "{temp}{format_units}".format(
-                    temp=temp,
-                    format_units=self.format_units
-                )
-                data.append(fmt_str)
-
-            output = "{format_prefix}{data}".format(
-                format_prefix=self.format_prefix,
-                data=self.temp_separator.join(data)
-            )
+                data.append(self.py3.safe_format(self.format_gpu_temp, {'temp': temp}))
 
+            output = "{data}".format(data=self.icon_separator.join(data))
```

Comment:
this bit still needs fixing like I described.

I've a friend coming over now so talk more another day